lucien-ios/AddNewComicBook/AddComicViewController.swift

```diff
@@ -181,6 +185,18 @@ final class AddComicViewController: UIViewController, UIPickerViewDelegate, UIPi
         deregisterFromKeyboardNotifications()
     }
 
+    @IBAction func addCoverButtonTapped(_ sender: UIButton) {
+        if AVCaptureDevice.authorizationStatus(for: AVMediaType.video) ==  AVAuthorizationStatus.authorized {
+            present(cameraViewController, animated: true, completion: nil)
+        } else {
+            AVCaptureDevice.requestAccess(for: AVMediaType.video, completionHandler: { (granted: Bool) -> Void in
+                if granted == true {
```

Comment:
Only optional Bools should be compared to true, in this case `if granted` will suffice